cumulusci.yml

```diff
@@ -296,11 +298,13 @@ plans:
                     - when: "'npsp' not in tasks.get_installed_packages()"
                       action: hide
                 ui_options:
+                    name: "Install Page Layouts (NPSP)"
```

Comment:
I've been using "Deploy xyz" for unmanaged plan steps on my side of the house. I don't feel strongly about standardizing that language, but I thought I'd mention it.